pyflakes/test/test_imports.py

```diff
@@ -1031,6 +1039,48 @@ def test_futureImportStar(self):
         from __future__ import *
         ''', m.FutureFeatureNotDefined)
 
+    def test_ignores_typing_imports(self):
```

Comment:
let's put these tests into the type_annotations test

Reply:
Done!